app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -337,6 +337,10 @@ def self.provision_class(via)
       end
     end
 
+    def self.default_blacklisted_event_names
+      Settings.ems.ems_vmware.blacklisted_event_names
```

Comment:
NOTE this one can't use ems_type because for the infra_manager ems_type is vmwarews not vmware